microservices/google-api/google_search.py

```diff
@@ -280,12 +280,14 @@ def build_service():
         rowlimit = 20000
         index = 0
 
-        def daterange(startDate, endDate):
+        def daterange(start_date, end_date):
             """yields a generator of all dates from startDate to endDate"""
-            logger.debug("daterange called with startDate: %s and endDate: %s", startDate, endDate)
-            assert endDate >= startDate, 'startDate cannot exceed endDate in daterange generator'
-            for n in range(int((endDate - startDate).days)+1):
-                yield startDate + timedelta(n)
+            logger.debug("daterange called with startDate: %s and endDate: %s",
+                         start_date, end_date)
+            assert end_date >= start_date, 'startDate cannot exceed endDate \
+             in daterange generator'
+            for _n in range(int((end_date - start_date).days)+1):
+                yield start_date + timedelta(_n)
 
         search_analytics_response = ''
 
```

Comment:
I'm going to initialize `date_in_range` here to an empty list in order to avoid pylint [undefined-loop-variable] warnings.
```
date_in_range = []
    for date_in_range in daterange(start_dt, end_dt):
```